### ISSUE TITLE
Can we restrict number of images to be picked

### ISSUE THREAD

**Issue by laxayyy** [yasirkula/UnityNativeGallery#246]
Does the plugin support to restrict the number of images to be picked when using GetImagesFromGallery.

For example - in android/iOS  file selector, restrict the input images to 10

**Comment by yasirkula** [comment]
It's possible only on iOS by changing the last parameter to 10: https://github.com/yasirkula/UnityNativeGallery/blob/62de2dc50ab85e57fafa105909247470155a4ad7/Plugins/NativeGallery/NativeGallery.cs#L625

**Comment by yashs-devstree** [comment]
Is there any way we can restrict them in Android also?

**Comment by yasirkula** [comment]
Unfortunately not.

**Comment by itkwtl1487** [comment]
Is there a solution now for Android?

**Comment by yasirkula** [comment]
NativeGallery still doesn't support it.

**Comment by yasirkula** [comment]
Of course: https://github.com/yasirkula/UnityNativeGallery/tree/master/.github/AAR%20Source%20(Android)
